experimental/builder/include/ck_tile/builder/conv_algorithm_limits.hpp

```diff
@@ -30,4 +31,25 @@ concept AccessOrderLimits = requires {
              (Value[2] >= 0 && Value[2] < 3));
 };
 
+// Limits for tile sizes
```

Comment:
Is this universal for all architectures (it looks like a generic mathematical condition)? Maybe add a short explanation why the conditions logically obvious or if there is a technical reference.